src/main/java/Duke/DukeSave.java

```diff
@@ -0,0 +1,96 @@
+package Duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+//import java.nio.file.Files;
+//import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class DukeSave {
+    /**
+     * Divider used to separate details of a task saved in file
+     */
+    private static final String SAVEFILESEPARATOR = ", ";
+    private static String filePath = "DukeTaskList.txt";
+    private static File textFile;
+    private static Scanner textFileScanner;
+
+    private static void setupFile() {
+        textFile = new File(filePath);
+
+        try {
+            textFileScanner = new Scanner(textFile);
+        } catch (FileNotFoundException e) {
+            System.out.println("I cannot seem to access the saved tasks. Did you perhaps lock it away?");
```

Comment:
For all your error messages, you might want to move them into a file as named constants, or using enum for it.